server/src/routes/api/v1/pastasRouter.js

```diff
@@ -1,38 +1,36 @@
-import express from 'express'
-import PastasSerializer from '../../../serializers/PastasSerializer.js'
-import pastaReviewsRouter from './pastaReviewsRouter.js'
+import express from "express";
+import PastasSerializer from "../../../serializers/PastasSerializer.js";
+import pastaReviewsRouter from "./pastaReviewsRouter.js";
 
-import { Pasta } from '../../../models/index.js'
+import { Pasta } from "../../../models/index.js";
 
-const pastasRouter = new express.Router()
+const pastasRouter = new express.Router();
 
-pastasRouter.get('/', async (req, res) => {
-    try {
-        const pastas = await Pasta.query()
-        const serializedPastas = await Promise.all(
-            pastas.map(async (pasta) => {
-                return PastasSerializer.getDetails(pasta)
-            })
-        )
-        return res.status(200).json({ pastas: serializedPastas })
-    } catch (error) {
-        return res.status(500).json({errors: error})
-    }
-})
+pastasRouter.get("/", async (req, res) => {
+  try {
+    const pastas = await Pasta.query();
+    const serializedPastas = await Promise.all(
+      pastas.map(async (pasta) => {
+        return PastasSerializer.getDetails(pasta);
```

Comment:
you are overusing your `getDetails` method. 

you should have one serialization method for the index page (like getSummaryForPastas) and one for the show (like getDetailsForPasta). By using one method for both endpoints, we are getting unnecessary info in your response, which defeats the purpose of the serializer!